server/src/main/scala/sbt/server/SbtClientHandler.scala

```diff
@@ -150,7 +150,7 @@ class SbtClientHandler(
           // Now clean out the request handler
           readLineRequests -= serial
           confirmRequests -= serial
-        case None => // TODO - error
+        case _ => // TODO - error
```

Comment:
if you're getting MatchErrors that's a logical bug. The compiler should be warning us of that.  It's fine for now, but generally, we should use _ unless we really do want to catch everything.